tools/hardhat-viem-example/.env.example

```diff
@@ -0,0 +1,23 @@
+# For Hedera Testnet
+# The operator private keys start with "0x" and are in hexadecimal format
+# Your testnet account ECDSA hex-encoded private key (Ex: 0xb46751179bc8aa9e129d34463e46cd...)
+TESTNET_OPERATOR_PRIVATE_KEY=0xb46751179bc8aa9e129d34463e46cd924055112eb30b31637b5081b56ad96129
```

Comment:
is this your personal account PK? If so, then you should leave this blank similar to previewnet and mainnet below.

Reply:
@acuarica This PR was initially a Fork (git@github.com:arianejasuwienas/hedera-viem-example-project.git) of this repository:
https://github.com/hashgraph/hedera-hardhat-example-project/blob/main/.env.example#L4

I've removed this key, since its origin was a unknown for me too.

Reply:
Sounds good. Thanks for pointing this out.